daemon/info.go

```diff
@@ -113,6 +113,25 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	return v, nil
 }
 
+// SystemVersion returns version information about the daemon.
+func (daemon *Daemon) SystemVersion() types.Version {
```

Comment:
this allows backends to change the information in the version. I don't know if it's recommendable, but I'm just following conventions :smile_cat: